Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.13-alpine as builder
+
+WORKDIR /build
+COPY . /build
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
+
+RUN apk --no-cache add make
+
+RUN make build-db
+
+FROM scratch
+COPY --from=builder /build/trivy.db.gz .
```

Comment:
How about this dockerfile.
Use ARG or ENV.  My preference is ARG.

```
FROM scratch
ARG DBTYPE
ADD https://github.com/aquasecurity/trivy-db/releases/latest/download/${DBTYPE}.db.gz .
```

### build 
Use trivy.db.gz

```
export DBTYPE=trivy
docker built -t ${DBTYPE}-db --build-arg DBTYPE=${DBTYPE} .
```

Use trivy-light.db.gz

```
export DBTYPE=trivy-light
docker built -t ${DBTYPE}-db --build-arg DBTYPE=${DBTYPE} .
```

Reply:
I'd like to avoid adding artifacts to the dockerfile that are volatile, if we can build it from scratch as well. If for some reason the github action pipeline breaks and the release doesn't get published anymore, users can still rebuild from scratch.

Reply:
Thank you very much.
I think it makes sense.

I have some requests. 
- I would recommend changing the CI to use Makefile
- I think it is better to include Clean target in Makefile.
- Please support trivy-light.db.gz target in Makefile.

Reply:
Thanks, I'll update the PR to include your requested changes. AFAIK, I have no option to test the CI pipeline though.

Reply:
So sorry, no options.

Reply:
I've added support for the clean target and the light db using DB_TYPE="light" as you suggested.
I'll need to take a look into the CI integration and figure out how to separate the fetching of github vuln db tarballs out.